### DISCUSSION TITLE
Trying to use an app

### DISCUSSION THREAD

**Discussion by JYANG2002** [nasa/cFS#423] [Q&A]
I'm still fairly new to cFS but let's say I cloned myself a checksum(cs) into my app directory in cFS and I updated the target and startup. How do I know my app is running and how can I use it.

**Comment by skliper** [comment]
https://github.com/OpenSatKit/OpenSatKit has an example of how to actually use cFS in a sample distribution.  If you're able to review the documentation within CS, see https://github.com/nasa/cFS-GroundSystem for a way to send "commands" and receive/interpret "telemetry".  Note CS is a little tricky since not all features are supported on all targets.  For example checksumming modules is only possible on systems where OSAL currently supports getting the module address and size.

More generally though - Apps typically report a startup message to the console when loaded successfully, for example:
`EVS Port1 66/1/CS 1: CS Initialized. Version 2.5.0.0`

You can also typically get the version information and see a command counter increment by sending a "Noop" command using cFS-GroundSystem/Cosmos/cmdutil or whatever tool you prefer.  Apps also generally provide a housekeeping message that contain operational information like a count of successful commands received, count of command errors, etc.
